components/o-forms/src/tsx/Select.tsx

```diff
@@ -0,0 +1,113 @@
+import {getInputClasses, uniqueId} from '../utils';
+import {FormError, InputProps, TypeFormField, FormField} from './Form';
+
+interface TypeSelect extends InputProps {
+	highlight?: 'valid' | 'invalid';
+	errorMessage?: string;
+	suffix?: JSX.Element;
+	isSmall?: boolean;
+	children?: JSX.Element[];
```

Comment:
I think this might need to also allow one element:
```suggestion
	children?: JSX.Element | JSX.Element[];
```